Tests/UITests/TestApp/TestAppTestingSetup.swift

```diff
@@ -0,0 +1,45 @@
+//
+// This source file is part of the Stanford Spezi open source project
+//
+// SPDX-FileCopyrightText: 2022 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Security
+import SwiftUI
+
+
+private struct HealthGPTAppTestingSetup: ViewModifier {
+    func resetKeychain() {
+        let secItemClasses = [
+            kSecClassGenericPassword,
+            kSecClassInternetPassword,
+            kSecClassCertificate,
+            kSecClassKey,
+            kSecClassIdentity
+        ]
+
+        for itemClass in secItemClasses {
+            let spec: [String: Any] = [kSecClass as String: itemClass]
+            SecItemDelete(spec as CFDictionary)
+        }
+    }
```

Comment:
Would make more senes to use something like this from SpeziStorage: https://github.com/StanfordSpezi/SpeziStorage/blob/b958df9b31f24800388a7bfc28f457ce7b82556c/Sources/SpeziSecureStorage/SecureStorage.swift#L225